pkg/api/validate/types.go

```diff
@@ -28,6 +28,9 @@ var (
 	// This regexp is to check image version format
 	imageVersion = regexp.MustCompile(`^[0-9]{3}.[0-9]{1,4}.[0-9]{8}$`)
 
+	// This regexp is to check cluster version (plugin) format
+	clusterVersion = regexp.MustCompile(`^v\d+\.\d+$`)
```

Comment:
We should start unit-testing these regexps. Can you add one here?

Reply:
added